index.html

```diff
@@ -56,7 +55,7 @@
 
       </section>
       <section class="main-section" id="Section_3">
-        <header>Section 3</header>
+        <h2>Section 3</h2>
```

Comment:
I suggest using hx (h1, h2, h3...) tags for your header titles, to add more semantic.